test/integration/puppetlabs/services/certificate_authority/certificate_authority_int_test.clj

```diff
@@ -53,24 +53,28 @@
         app
         {:certificate-authority {:certificate-status
                                  {:client-whitelist ["localhost"]}}
-         :authorization         {:rules [{:match-request
-                                                 {:path "/puppet-ca/v1/certificate"
-                                                  :type "path"}
-                                          :allow ["nonlocalhost"]}]}}
-        (testing "are allowed"
-          (doseq [ca-mount-point ca-mount-points
-                  endpoint ["certificate_status/localhost"
-                            "certificate_statuses/ignored"]]
-            (testing (str "for the " endpoint " endpoint")
-              (let [response (http-get (str ca-mount-point endpoint))]
-                (is (= 200 (:status response))
-                    (ks/pprint-to-string response))))))
-        (testing "are denied when denied by rule to the certificate endpoint"
-          (doseq [ca-mount-point ca-mount-points]
-            (let [response (http-get (str ca-mount-point
-                                          "certificate/localhost"))]
-              (is (= 403 (:status response))
-                  (ks/pprint-to-string response)))))))))
+         :authorization         {:version 1
+                                 :rules [{:match-request
+                                          {:path "/puppet-ca/v1/certificate"
+                                           :type "path"}
+                                          :allow ["nonlocalhost"]
+                                          :sort-order 1
+                                          :name "cert"}]}}
+        (logutils/with-test-logging
```

Comment:
Sure. FYI there's already a top-level `with-test-logging` that'll swallow those things, but I could push it down further